### DISCUSSION TITLE
Dbeaver Load SchemaCache takes forever

### DISCUSSION THREAD

**Discussion by nixland** [dbeaver/dbeaver#19880] [Q&A]
Hi,
Please help. Starting today, when I run any query in Dbeaver in 1 particular database (Sql Server), the background process "Load SchemaCache" takes forever a.k.a. never finish. 
The rest of Sql Server databases on Dbeaver have no problem at all when running queries.
I tried to query that particular database using Microsoft SSMS using same query and there is no problem there.
I've tried to restart my laptop and the problem still exists.

Any idea what is the problem? Is there any cache files I should clear or delete in Dbeaver folders, or else ?

Thanks

**Comment by nixland** [comment]
I just found the root cause. 
In that query script I wrote lots of queries from many database and tables.
I tried to create new query script in the same database, and run couple of queries and it runs well without any problem.
So it seems that Dbeaver tries to load all schema from all database and tables in that problematic query script into the cache, and Dbeaver can not handle it. But previously it has no problem.
It's just my assumption though.

**Reply by ammarsaf** [reply]
It happened to me right now. How do you solve it? Just close any script that query too much table and create new simple script?

**Reply by ammarsaf** [reply]
anything to solve this @dBeaverZach
